tests/conftest.py

```diff
@@ -342,6 +296,82 @@ def _create_task(task_name=""):
         yield _create_task
 
 
+@pytest.fixture(scope="module")
+def persisted_task(request, system):
+    """Gets the persisted task based on the task name."""
+    task_name = _get_marker_value(request, "task_name")
+
+    if task_name in system.tasks.task_names:
+        return system.tasks[task_name]
+
+    pytest.skip(
+        "Could not detect a persisted task that has the given name."
+        "Cannot proceed to run tests. Import the NI MAX configuration file located at "
+        "nidaqmx\\tests\\max_config\\nidaqmxMaxConfig.ini to create the required tasks."
+    )
+    return None
+
+
+@pytest.fixture(scope="module")
+def persisted_scale(request, system):
+    """Gets the persisted scale based on the scale name."""
+    scale_name = _get_marker_value(request, "scale_name")
+    if scale_name in system.scales:
+        return system.scales[scale_name]
+    pytest.skip(
+        "Could not detect a persisted scale with the requested scale name.  Cannot proceed "
+        "to run tests. Import the NI MAX configuration file located at "
+        "nidaqmx\\tests\\max_config\\nidaqmxMaxConfig.ini to create the required scales."
+    )
+    return None
+
+
+@pytest.fixture(scope="module")
+def persisted_channel(request, system):
+    """Gets the persisted channel based on the channel name."""
+    channel_name = _get_marker_value(request, "channel_name")
+
+    if channel_name in system.global_channels.global_channel_names:
+        return system.global_channels[channel_name]
+
+    pytest.skip(
+        "Could not detect a global channel that has the given name."
+        "Cannot proceed to run tests. Import the NI MAX configuration file located at "
+        "nidaqmx\\tests\\max_config\\nidaqmxMaxConfig.ini to create the required channels."
+    )
+    return None
+
+
+@pytest.fixture(scope="function")
+def watchdog_task(
+    request, any_x_series_device, generate_watchdog_task
+) -> nidaqmx.system.WatchdogTask:
+    """Gets a watchdog task instance."""
+    # set default values used for the initialization of the task.
+    device_name = _get_marker_value(request, "device_name", any_x_series_device.name)
+    timeout = _get_marker_value(request, "time_out", 0.5)
```

Comment:
`time_out` should not have an underscore.
```suggestion
    timeout = _get_marker_value(request, "timeout", 0.5)
```

Reply:
This is not fixed. The `nidaqmx` module uses the name `timeout`. There are 0 occurrences of the name `time_out`.
```
PS D:\dev\nidaqmx-python> rg -c timeout .\generated\nidaqmx\
.\generated\nidaqmx\_task_modules\triggering\reference_trigger.py:1
.\generated\nidaqmx\_task_modules\out_stream.py:20
.\generated\nidaqmx\_task_modules\in_stream.py:28
.\generated\nidaqmx\_stubs\session_pb2_grpc.py:10
.\generated\nidaqmx\_stubs\nidaqmx_pb2_grpc.py:778
.\generated\nidaqmx\_stubs\nidaqmx_pb2.pyi:156
.\generated\nidaqmx\_stubs\nidaqmx_pb2.py:1
.\generated\nidaqmx\_library_interpreter.py:101
.\generated\nidaqmx\_grpc_interpreter.py:104
.\generated\nidaqmx\_base_interpreter.py:52
.\generated\nidaqmx\task.py:31
.\generated\nidaqmx\system\watchdog.py:8
.\generated\nidaqmx\system\system.py:7
.\generated\nidaqmx\system\device.py:3
.\generated\nidaqmx\stream_writers.py:210
.\generated\nidaqmx\stream_readers.py:234
.\generated\nidaqmx\constants.py:1
PS D:\dev\nidaqmx-python> rg -c time_out .\generated\nidaqmx
PS D:\dev\nidaqmx-python>
```

The tests should use the same spelling as the `nidaqmx` package, not the alternate spelling `time_out`.

Reply:
This is the last open issue on this PR, so I updated the code to use `timeout`.